build.sh

```diff
@@ -12,6 +12,9 @@ if [[ $arcadeLine =~ $pattern ]]; then
 fi
 echo "Found 2.x SDK $SDK_VERSION, 3.x SDK $SDK3_VERSION, Arcade bootstrap $ARCADE_BOOTSTRAP_VERSION"
 
+echo "IP information:"
+ifconfig -a
```

Comment:
ifconfig may not exist on all distributions so we should ignore errors here. (it was replaced by ip utility)